src/NzbDrone.Core/Notifications/Boxcar/BoxcarSettings.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using FluentValidation;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.Notifications.Boxcar
+{
+    public class BoxcarSettingsValidator : AbstractValidator<BoxcarSettings>
+    {
+        public BoxcarSettingsValidator()
+        {
+            RuleFor(c => c.Token).NotEmpty();
+        }
+    }
+
+    public class BoxcarSettings : IProviderConfig
+    {
+        private static readonly BoxcarSettingsValidator Validator = new BoxcarSettingsValidator();
+
+        [FieldDefinition(0, Label = "Access Token", HelpLink = "https://boxcar.io/client")]
```

Comment:
Needs HelpText with information on where to get the key from (Settings in the iOS app), the HelpLink isn't useful here.